File: dotNet-sdk/Semantria/Mapping/Output/DocPhrase.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Xml.Serialization;

namespace Semantria.Com.Mapping.Output
{
    [DataContract(Name = "phrase")]
    [XmlRootAttribute("phrase", Namespace = "")]
    public class DocPhrase
    {
        [DataMember(Name = "negating_phrase")]
        [XmlElementAttribute("negating_phrase")]
        public string NegatingPhrase { get; set; }

        [DataMember(Name = "is_negated")]
        [XmlElementAttribute("is_negated")]
        public bool IsNegated { get; set; }

        [DataMember(Name = "sentiment_score")]
        [XmlElementAttribute("sentiment_score")]
        public float SentimentScore { get; set; }

        [DataMember(Name = "title")]
        [XmlElementAttribute("title")]
        public string Title { get; set; }
    }
}

File: java-sdk/com/semantria/objects/output/DocPhrase.java
package com.semantria.objects.output;

import javax.xml.bind.annotation.XmlElement;

public class DocPhrase {
    private String title;
    private Float sentiment_score;
    private String negating_phrase;
    private Boolean is_negated;

    @XmlElement(name="title")
    public String getTitle() { return title; }
    @XmlElement(name="sentiment_score")
    public Float getSentimentScore() { return sentiment_score; }
    @XmlElement(name="negating_phrase")
    public String getNegatingPhrase() { return negating_phrase; }
    @XmlElement(name="is_negated")
    public Boolean getIsNegated() { return is_negated; }

    public void setTitle(String title) {this.title = title;}
    public void setSentimentScore(Float sentiment_score) {this.sentiment_score = sentiment_score;}
    public void setNegatingPhrase(String negating_phrase) {this.negating_phrase = negating_phrase;}
    public void setIsNegated(Boolean is_negated) {this.is_negated = is_negated;}
}
